tests/utils_tests/test_autoreload.py

```diff
@@ -168,7 +168,7 @@ def test_common_roots(self):
 class TestSysPathDirectories(SimpleTestCase):
     def setUp(self):
         self._directory = tempfile.TemporaryDirectory()
-        self.directory = Path(self._directory.name).resolve().absolute()
+        self.directory = Path(self._directory.name).resolve(strict=True).absolute()
```

Comment:
I'm not sure it's really necessary to use `strict=True` after `tempfile.TemporaryDirectory()` as it should raise if the path cannot be created. Not that it really hurts.

Reply:
I think I'm happy with it. It declares the intention.